clang/include/clang/Basic/arm_sve_sme_incl.td

```diff
@@ -225,7 +225,7 @@ def IsStreamingCompatible           : FlagType<0x4000000000>;
 def IsReadZA                        : FlagType<0x8000000000>;
 def IsWriteZA                       : FlagType<0x10000000000>;
 def IsReductionQV                   : FlagType<0x20000000000>;
-def IsStreamingOrSVE2p1             : FlagType<0x40000000000>; // Use for intrinsics that are common between sme/sme2 and sve2p1.
+def IsSVEOrStreamingSVE             : FlagType<0x40000000000>; // Use for intrinsics that are common between SVE and SME.
```

Comment:
If you permit a bit of bike shedding I don't think this is a good name.  From what I can see the new flag is used to trigger dynamic resolution to determine if the builtin is available to use based on the target features along with any keywords associated with the function.  Perhaps `RequiresDynamicVerification`? or ideally something shorter that has the same meaning.